### PR TITLE
Add the iOS Saving Links article as a default read for new Omnivore users on iOS

### PR DESCRIPTION
**Pull request by jacksonh**
This also re-orders things a bit, making sure the getting started
guide is always the topmost in a user's library.

### CODE REVIEW

File: packages/api/src/services/popular_reads.ts [hunk 1]
```diff
@@ -142,6 +141,9 @@ export const addPopularReadsForNewUser = async (
       break
   }
 
+  // We always want this to be the top-most article in the user's
+  // list. So we save it last to have the greatest saved_at
+  defaultReads.push('omnivore_get_started')
```
Comment (sywhb) [inline packages/api/src/services/popular_reads.ts:146]: Nice

**Review by sywhb — APPROVED** [review]
LGTM 👍

**Comment by sywhb** [comment]
@jacksonh We also need to update the test because now the ios guide is available

**Comment by jacksonh** [comment]
eek tests! Thanks, updating now.

**Pull request merged**